src/sentry/uptime/subscriptions/subscriptions.py

```diff
@@ -294,7 +323,10 @@ def disable_project_uptime_subscription(uptime_monitor: ProjectUptimeSubscriptio
     if uptime_monitor.status == ObjectStatus.DISABLED:
         return
 
-    uptime_monitor.update(status=ObjectStatus.DISABLED)
+    uptime_monitor.status = ObjectStatus.DISABLED
+    uptime_monitor.save()
```

Comment:
Same here - update is the same as what you're doing here

Reply:
See my other comment